iotdb-core/node-commons/src/main/java/org/apache/iotdb/commons/udf/builtin/UDTFAbs.java

```diff
@@ -89,6 +97,14 @@ public Object transform(Row row) throws IOException {
         return Math.abs(row.getFloat(0));
       case DOUBLE:
         return Math.abs(row.getDouble(0));
+      case VECTOR:
+      case UNKNOWN:
```

Comment:
Should we add VECTOR and UNKNOWN types to other switch case statements?

Reply:
Removed.